tests/ci/clickhouse_helper.py

```diff
@@ -109,6 +109,16 @@ def _insert_post(*args, **kwargs):
     def _insert_json_str_info(self, db, table, json_str):
         self.insert_json_str(self.url, self.auth, db, table, json_str)
 
+    def insert_json_into(self, db, table, json_str, safe=True):
```

Comment:
it's 100% the same as `ClickHouseHelper.insert_event_into`, but you manually do `json_str = json.dumps(data)`. Let's use `insert_event_into`.